elements/howto-menu/howto-menu.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function() {
+  /**
+   * Define key codes to help with handling keyboard events.
+   */
+  const KEYCODE = {
+    DOWN: 40,
+    ESCAPE: 27,
+    TAB: 9,
+    UP: 38,
+  };
+
+  class HowtoMenu extends HTMLElement {
+    /**
+     * A getter for the first child which is a menuitem.
+     */
+    get _firstMenuItem() {
```

Comment:
I think getters should only be used if they run in O(1). But I am happy to put that up for discussion.